### ISSUE TITLE
Date picker NVDA accessibility issue

### ISSUE THREAD

**Issue by robertjmccarthy** [ministryofjustice/moj-frontend#1109]
## What
Document the accessibility issue for the component on the site, resolve the identified issue, implement the update, and notify users accordingly.

## Why
Slack message on support channel flagged an accessibility issue raised: https://mojdt.slack.com/archives/CH5RUSB27/p1737123944815689

## Who needs to be involved
- Accessibility Specialist (leading)
- Developer (supporting)

## Done
When a resolution is found, applied and communicated out to users (and specifically communicated to the team that raised the issue)

**Comment by benproctor-rogers** [comment]
Currently investigating routes for testing with NVDA on MoJ devices. Will update with results of investigation by end of 01/25 at the latest. Once we are able to test using this screen reader we can work on a fix and hopefully avoid NVDA related issues in future.

**Comment by Whersi** [comment]
@helennickols [Accessibility notices for components with known issues](https://docs.google.com/document/d/1r_adg3ByOFahh3jV_c__XirD3Z7osmbm/edit?usp=sharing&ouid=116961598388358760611&rtpof=true&sd=true)

**Comment by helennickols** [comment]
I've done the content work on these statements. I just need a date from the devs.

**Comment by helennickols** [comment]
Fix date confirmed as April. This is ready to be added.

**Comment by murrlipp** [comment]
Changes made and now live.

**Comment by robertjmccarthy** [comment]
Accessibility issue information added to component page.

### To be done
- [ ] Fix found and applied to component
- [ ] Information to be removed from component once fix applied

**Comment by benproctor-rogers** [comment]
Initial testing with NVDA (via Assistiv labs) is complete. 

When switching from browsing mode to focus mode the issue no longer occurs and the focus indicator on each date is displayed correctly. In my mind, this is how NVDA should work. I shall speak to User Vision (the audit contractor) to discuss this and whether they think that NVDA users would know to do this. I'll update this ticket once I've had that conversation.

**Comment by benproctor-rogers** [comment]
I have spoken to UserVision. In their experience of working with NVDA users they advise that most users would not know to manually switch between browse mode and focus mode and that we should come up with a fix for this. They have a suggestion and I can talk to @chrispymm about this. 
We are working on a permanent in house NVDA testing solution to hopefully avoid this in future.

**Comment by benproctor-rogers** [comment]
I have checked the W3C example Date Picker. It does not have the NVDA issue.

I spoke to User Vision (the MoJ third party contractor) and they highlighted that two services have used the Date Picker and did not have this issue. These were "Book a prison visit" and "Bichard". I shall investigate these further to see what version of the Date Picker was used.

**Comment by helennickols** [comment]
The date has been changed to August 2025. @benproctor-rogers is this ready to work on or are we awaiting more info?
